test/PrintAsObjC/imports.swift

```diff
@@ -5,10 +5,16 @@
 // RUN: %target-swift-frontend(mock-sdk: %clang-importer-sdk) -I %S/Inputs/custom-modules/ -F %S/Inputs/ -parse-as-library %t/imports.swiftmodule -typecheck -emit-objc-header-path %t/imports.h -import-objc-header %S/../Inputs/empty.h -disable-objc-attr-requires-foundation-module
 // RUN: %FileCheck %s < %t/imports.h
 // RUN: %FileCheck -check-prefix=NEGATIVE %s < %t/imports.h
-// RUN: %check-in-clang %t/imports.h -I %S/Inputs/custom-modules/ -F %S/Inputs/
+// RUN: %check-in-clang %t/imports.h -I %S/Inputs/custom-modules/ -F %S/Inputs/ -Watimport-in-framework-header
```

Comment:
Did you verify that the warning fires without your fix here? I think it really does have to look like a framework header for it to show up. We should make sure we're testing the right thing.

Reply:
You're right, clang looks up path it's compiling for something with a .framework extension. I'll check if one of the other tests is more suitable.

Reply:
It's fine to add a new RUN line, too; sometimes that's lighter weight than adding a whole new test.